app/lib/services/restApi.test.ts

```
@@ -39,9 +39,21 @@ jest.mock('react-native-device-info', () => {
 	};
 });
 
-function loadRegisterPushToken(platform: 'ios' | 'android' = 'android') {
+function loadRegisterPushToken(
+	platform: 'ios' | 'android' = 'android',
+	mockServerVersion = '8.0.0'
+) {
```

ESLint

[error] Replace `⏎↹platform:·'ios'·|·'android'·=·'android',⏎↹mockServerVersion·=·'8.0.0'⏎` with `platform:·'ios'·|·'android'·=·'android',·mockServerVersion·=·'8.0.0'`

(prettier/prettier)